icalevents/icalparser.py

```diff
@@ -284,42 +399,40 @@ def parse_rrule(component, tz=UTC):
         if not isinstance(rrules, list):
             rrules = [rrules]
 
-        # Since DTSTART are always made timezone aware, UNTIL with no tzinfo
-        # must be converted to UTC.
-        for rule in rrules:
-            until = rule.get("until")
-            for idx, dt in enumerate(until or []):
-                if not hasattr(dt, 'tzinfo'):
-                    until[idx] = normalize(normalize(dt, tz=tz), tz=UTC)
+        # If dtstart is a datetime, make sure it's in a timezone.
+        rdtstart = component['dtstart'].dt
+        if type(rdtstart) is datetime:
+            rdtstart = normalize(rdtstart, tz=tz)
 
         # Parse the rrules, might return a rruleset instance, instead of rrule
-        rule = rrulestr('\n'.join(x.to_ical().decode() for x in rrules), dtstart=normalize(component['dtstart'].dt, tz=tz))
-        
+        rule = rrulestr('\n'.join(x.to_ical().decode() for x in rrules),
+                        dtstart=rdtstart)
+
         if component.get('exdate'):
             # Make sure, to work with a rruleset
             if isinstance(rule, rrule):
                 rules = rruleset()
                 rules.rrule(rule)
                 rule = rules
-            
+
             # Add exdates to the rruleset
             for exd in extract_exdates(component):
                 rule.exdate(exd)
-        
-        #TODO: What about rdates and exrules?
-        
+
+        # TODO: What about rdates and exrules?
```

Comment:
Is this something we should worry about?

Reply:
Dunno. I don't think that's my comment, I must have just put the space in reflexively.